acceptance/tests/pxp-module-puppet/negative/attempt_puppet_flag_whitelist_bypass.rb

```diff
@@ -6,6 +6,7 @@
 
   step 'pick test host' do
     @test_host = agents.first
+    skip_test 'Test not windows compatible' if @test_host.platform =~ /windows/
```

Comment:
@joshcooper Sorry for the churn.  Seems like the best solution is to remove the confine.  I'll make a new PR.